### PR TITLE
Refactor/cc 0150 refactor admin sidebar

### PR DESCRIPTION
**Pull request by Double-Twelve**
image

### CODE REVIEW

**Review by GLC-coder — COMMENTED** [review]
Well done! Shawn

File: src/components/SideBar/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { Dispatch, SetStateAction } from "react";
+import { Text, Flex, Image, Icon, Tooltip, Box } from "@chakra-ui/react";
```
Comment (zhuocun) [inline src/components/SideBar/Logo.tsx:2]: The box is an unused import :)

File: src/interfaces/navigationItem.ts [hunk 2]
```diff
@@ -1,6 +1,9 @@
-export interface ISideBarItem {
+import { IconType } from "react-icons";
+
+export interface ISidebarItem {
```
Comment (zhuocun) [inline src/interfaces/navigationItem.ts:3]: Maybe either the name of the file or the interface can be changed to that of the other to reach consistency :)

**Review by ShawnWang86b — APPROVED** [review]
very good tooltip

**Review by GLC-coder — APPROVED** [review]
LGTM

**Pull request merged**